Carmunication/Controllers/SessionList/CustomListTableViewCell.swift

```diff
@@ -4,47 +4,167 @@
 //
 //  Created by 김동현 on 2023/09/24.
 //
-
+import SnapKit
 import UIKit
 
-import SnapKit
+// MARK: - Preview canvas 세팅
```

Comment:
아카데미 코딩 컨벤션에는 내장, 서드파티 상관없에 ABC 순으로 작성하지만,

프레임워크 임포트가 많아질 경우 보기 불편하니, 내장 프레임워크랑 서드파티 프레임워크를 나눠서 작성하자는 얘기를 나눴었어요 !

우선순위가 높은 수정은 아니니 천천히 수정하셔도 될 듯 합니다 :)

```suggestion
//
//  Created by 김동현 on 2023/09/24.
//

import UIKit

import SnapKit

// MARK: - Preview canvas 세팅
```

아래 노션에 작성해뒀으니 확인 부탁드릴게요 !
https://www.notion.so/bef384f498264cf59a910840c71b5105?pvs=4

Reply:
제가 까먹고 있었군요
다음 이슈에 반영토록 하겠습니다.